packages/core/src/batch/BatchSystem.js

```diff
@@ -66,4 +66,61 @@ export class BatchSystem extends System
     {
         this.setObjectRenderer(this.emptyRenderer);
     }
+
+    /**
+     * Handy function for batch renderers: copies bound textures in first maxTextures locations to array
+     * sets actual _batchLocation for them
+     *
+     * @param arr
+     * @param maxTextures
+     */
+    copyBoundTextures(arr, maxTextures)
+    {
+        const { boundTextures } = this.renderer.texture;
+
+        for (let i = maxTextures - 1; i >= 0; --i)
+        {
+            arr[i] = boundTextures[i] || null;
+            if (arr[i])
+            {
+                arr[i]._batchLocation = i;
+            }
+        }
+    }
+
+    boundArray(texArray, boundTextures, batchId, maxTextures)
```

Comment:
Documentation for this method? These docs are very helpful fir us converting to TypeScript